ProgrammingFundamentals/Session1/Assignment2/Assignment2.java

```diff
@@ -0,0 +1,35 @@
+import java.util.*;
+/**
+ *
+ *
+ *	@author  Vinod Patel
+ *
+ */
+public class Assignment2 {
+	// Removing duplicate elements from Array
+	public int[] removeDuplicate(int input[]) {
+		ArrayList<Integer> al = new ArrayList<Integer>();//creating dynamic array to store non duplicate elements
+		
+		for(int i=0; i<input.length; i++) {
+			boolean flag=true;//checking non duplicacy of an element in arrays
+			for(int j=0; j<al.size(); j++) {
+				if(input[i] == (int)al.get(j)) {
+					flag=false;

```

Comment:
Give meaning fullname to variable. For example you can use isDuplicateFound rather than flag here. Naming of variables,classname, method name is important part in project work. Make sure names are meaningful in rest of your assignments.
Best Practices:
1. Indentation should be proper.
2. Give proper comment if code is complex.
3. Give meaningful name.
4. try to use less variable for best use of memory.